<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProntoSoccorso</class>
 <widget class="QWidget" name="ProntoSoccorso">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(230, 235, 239);</string>
  </property>
  <widget class="QLabel" name="InformazioniProfilo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>730</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(32, 32, 32);

</string>
   </property>
   <property name="text">
    <string>Modifica infermiere</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_Nome">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color: rgb(230, 235, 239);
color: rgb(0, 0, 0);
border: 0px solid black;
border-radius: 6px;

}
                            
QLineEdit:focus {
border: 1px solid rgb(18, 89, 156);
}</string>
   </property>
   <property name="maxLength">
    <number>30</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string notr="true">Nome</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_Cognome">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color: rgb(230, 235, 239);
color: rgb(0, 0, 0);
border: 0px solid black;
border-radius: 6px;

}
                            
QLineEdit:focus {
border: 1px solid rgb(18, 89, 156);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>30</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string notr="true">Cognome</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_CodiceFiscale">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color: rgb(230, 235, 239);
color: rgb(0, 0, 0);

border-radius: 6px;
}

QLineEdit:focus {
border: 1px solid rgb(18, 89, 156);
}
                 

</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="inputMask">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string notr="true">Codice Fiscale</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_Username">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color: rgb(230, 235, 239);
color: rgb(0, 0, 0);
border: 0px solid black;
border-radius: 6px;

}
                            
QLineEdit:focus {
border: 1px solid rgb(18, 89, 156);
}</string>
   </property>
   <property name="maxLength">
    <number>30</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string notr="true">Username</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_Password">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color: rgb(230, 235, 239);
color: rgb(0, 0, 0);
border: 0px solid black;
border-radius: 6px;

}
                            
QLineEdit:focus {
border: 1px solid rgb(18, 89, 156);
}</string>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string notr="true">Password</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonAnnulla">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>380</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(230, 235, 239);
	color: rgb(32, 32, 32);
border-radius: 10px;
}
QPushButton:hover {
background-color: rgb(199, 199, 199);

    }
</string>
   </property>
   <property name="text">
    <string>Annulla</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonSalva">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(230, 235, 239);
	color: rgb(18, 89, 156);
border-radius: 10px;
}
QPushButton:hover {
background-color: rgb(199, 199, 199);
    }
</string>
   </property>
   <property name="text">
    <string>Salva</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="LogoInfermieri">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../../../MATERIALE/IconaInfermiere.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateEdit {
setPlaceholderText:(&quot;Inserisci qui il testo...&quot;);
background-color: rgb(230, 235, 239);
color: rgb(0, 0, 0);
border: 0px solid black;
}
QDateEdit::up-button{
        width: 0px;
}
 QDateEdit::down-button {
width: 0px;
}
QDateEdit:focus{
border: 1px solid rgb(18, 89, 156);
}</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="displayFormat">
    <string>dd/MM/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>290</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(134, 134, 134);
color: rgb(18, 89, 156);</string>
   </property>
   <property name="text">
    <string>Dati personali dell'infermiere</string>
   </property>
  </widget>
  <widget class="QLabel" name="Sheda">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>290</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 235, 239);
border: 1px solid rgb(172, 172, 172);
border-radius: 5px;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Sheda_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>290</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 235, 239);
border: 1px solid rgb(172, 172, 172);
border-radius: 5px;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>290</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(134, 134, 134);
color: rgb(18, 89, 156);</string>
   </property>
   <property name="text">
    <string>Credenziali dell'infermiere nel sistema</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
background-color: rgb(230, 235, 239);
color: rgb(0, 0, 0);
border: 0px solid black;
border-radius: 6px;

}

QComboBox::drop-down {
image: url(:/newPrefix/UI UFFICIALI/MATERIALE/freccia.png);
width: 20px; 
height: 20px;
padding: 5px;

}

QComboBox QAbstractItemView {
/* Personalizza il menu a discesa */
        /* Definire le altre proprietà di stile per il menu a discesa qui */

border-radius: 6px;
        
    }
</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string>Uomo</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="maxVisibleItems">
    <number>3</number>
   </property>
   <property name="iconSize">
    <size>
     <width>14</width>
     <height>14</height>
    </size>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>Uomo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Donna</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Altro...</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 235, 239);
color: rgb(115, 117, 119);

border-radius: 6px;
   

</string>
   </property>
   <property name="text">
    <string>Data di nascita :</string>
   </property>
  </widget>
  <zorder>Sheda_2</zorder>
  <zorder>Sheda</zorder>
  <zorder>LogoInfermieri</zorder>
  <zorder>InformazioniProfilo</zorder>
  <zorder>LineEdit_Nome</zorder>
  <zorder>LineEdit_CodiceFiscale</zorder>
  <zorder>LineEdit_Cognome</zorder>
  <zorder>LineEdit_Username</zorder>
  <zorder>LineEdit_Password</zorder>
  <zorder>ButtonAnnulla</zorder>
  <zorder>ButtonSalva</zorder>
  <zorder>dateEdit</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>comboBox</zorder>
  <zorder>label_3</zorder>
 </widget>
 <resources>
  <include location="../../../../../../../asa.qrc"/>
 </resources>
 <connections/>
</ui>
